runtime/tests/third_party/dummy_device.toml

```diff
@@ -91,6 +58,38 @@ matrix = [
         "MultiControlledX",
```

Comment:
My other comment is about the format mismatch between the native gates and the other types, here we have strings in a list and below the gates are listed "at the top level" so to speak. Not sure what the TOML format allows, but I would be nice if this was uniform for gates (and even observables and measurements?)

Is the following allowed (to keep with the named gate format below)?
```py
[operators.gates.matrix]
MultiControlledX
```

Reply:
Yes, it is allowed, will do.

Reply:
Done